### DISCUSSION TITLE
Cutoff Voltage Determinations

### DISCUSSION THREAD

**Discussion by nbiju1998** [pybamm-team/PyBaMM#2626] [General]
Hello,

I had some general questions about how the max and min voltages are determined in the simulation. In the parameter list, I notice that the upper and lower cutoff voltages are set to be 4.1 V and 3.105 V, respectively. However, when I apply an extremely small discharge of 1/100C, the initial voltage is slightly above 3.8 V and goes until about 3.4 V. The total discharge capacity of the cell is 0.68 Ah which matches what has been specified. 

So why don't the voltages match up to the discharged capacity?

I am running the DFN model with the parameters from Marquis2019. This is what I am getting right now:

![image](https://user-images.githubusercontent.com/41459132/214994174-6582918a-eaaf-4afb-a4e1-4edcd89214d8.png)

**Comment by 1836005678** [comment]
My simulation using parameter set Marquis2019 did not show similar phenomenon. Instead, it's discharge capacity is about 1.2 times the nominal capacity at 1C. Please provide more details such as your codes.

**Reply by 1836005678** [reply]
I also use DFN model but i did not consider thermal physics.
